README.md

```diff
@@ -134,6 +134,9 @@ For `labelRenderer`, you can provide a full path - [see this PR](https://github.
 - `shouldExpandNode: function(keyName, data, level)` - determines if node should be expanded (root is expanded by default)
 - `hideRoot: Boolean` - if `true`, the root node is hidden.
 - `sortObjectKeys: Boolean | function(a, b)` - sorts object keys with compare function (optional). Isn't applied to iterable maps like `Immutable.Map`.
+- `onNodeExpansionChanging: function(keyName, data, level, expanded)` - invoked when a node is expanding or collapsing.
+- `onNodeExpansionChanged: function(keyName, data, level, expanded)` - invoked when a node is expanded or collapsed.
+- `isNodeExpansionDynamic: Boolean` - if `true`, `shouldExpandNode` will be called each render to determine the expansion state of a node, rather than just once during mount
```

Comment:
If you need it to be called on each render, you can create a new callback every time, for example:
```jsx
shouldExpandNode={this.handleShouldExpandNode.bind(this)}
```
instead of:
```jsx
shouldExpandNode={this.handleShouldExpandNode}
```
(considering `handleShouldExpandNode` is declared as a class prop)

It might be not that obvious (and should be mentioned in docs), but I think it's fair - `shouldExpandNode` is considered a pure function, and if it's not - just wrap it in a new one.